po/wesnoth-sof/wesnoth-sof.pot

```diff
@@ -3008,9 +3013,9 @@ msgstr ""
 #. cavalier = his unit type
 #: data/campaigns/Sceptre_of_Fire/scenarios/7_Outriding_the_Outriders.cfg:595
 msgid ""
-"I’m a cavalier, with Haldric II’s personal bodyguard. I was sent on a "
-"mission in the Northlands, and now elvish horsemen are chasing me. I barely "
-"evaded them."
+"I’m a cavalier with Haldric II’s personal bodyguard. I was sent on a mission "
+"in the Northlands, and now elvish horsemen are chasing me. I barely evaded "
+"them."
```

Comment:
Using pofix was suggested in the forum thread, but it doesn't seem capable of the task.

One known issue is illustrated with the line that I'm commenting on here. Pofix, as it currently exists, works on a line-by-line basis, so using Pofix for this one would be three lines with a high probability of false positives on `"them."`, and an absolute dependence on the translators' tools' word-wrap settings.

Yes, it could be adapted to fix that specific issue, which would reduce the number of strings that need to be added to it to around 120 for this release alone. My gut feeling is that, if I try to do it with Pofix, I'm going encounter more issues; so my question to anyone suggesting continuing with that tool is to ask "have you thought through the problems, rather than just saying that we already have a tool?".

Reply:
If pofix.py handles the stuff for about 90% of the cases that is already a huge win especially since it works around any stupidity / old files on the translators side. And yes, it will not be a 100% solution but better than trying to fix things manually and after getting the next update seeing that it was a moot try.

Reply:
What would make it a moot try? Is it stuff that would be helpful to consider even when using pofix?

Reply:
(I may be missing something but it looks like you would rewrite "evaded them." to "them." not the other way around.)

Certainly pofix is not perfect and can surely be made more robust or even a new tool could be made to do a better job. It just was enough so far that nobody bothered. I wouldn't expect big problems after 10 years of use but certainly hundreds of string changes in a stable version were not done before (and is the real issue here).

I think with "moot try" Ivanovic means changes done by hand (or just once anyway) that will be overwritten the next time someone sends in a translation update based on old po files.